portal/server/controllers/dashboard/filters/helpers.js

```diff
@@ -22,7 +22,8 @@ const submittedFiltersArray = (allSubmittedFilters = {}) => {
       const submittedValue = submittedFilters[field];
       const fieldHasMultipleValues = Array.isArray(submittedValue);
 
-      if (submittedValue) {
+      // NOTE: some filter values can have a value of false.
```

Comment:
what other values do we expect here?

Reply:
It will either be a string (could be any string value), true or false. If it's false then `if(submittedValue)` fails because it's javascript, so have to add an "or if value is false".

Added the comment so it's clear, does it make sense?